api/v3/utils.php

```diff
@@ -1442,10 +1442,20 @@ function _civicrm_api3_custom_data_get(&$returnArray, $checkPermission, $entity,
       //not standard - but some api did this so guess we should keep - cheap as chips
       $returnArray[$key] = $val;
 
-      // Shim to restore legacy behavior of ContactReference custom fields
+      // Shim to restore legacy behavior of ContactReference custom fielsd
```

Comment:
I feel like the spelling was correcter beforz :)

Reply:
ctrl+t for the winz!